### DISCUSSION TITLE
QueueCheck is a thing?

### DISCUSSION THREAD

**Discussion by BenceBakos** [spatie/laravel-health#171] [Q&A] [answered]
Hey, 
looks like there is some functionlaity in the documentation:
https://spatie.be/docs/laravel-health/v1/available-checks/queue

Wich is not in the code anymore. Shouldn't we delete the relevant part from the documentation? Or I miss something?

**Comment by freekmurze** [comment, marked as answer]
It still exists: https://github.com/spatie/laravel-health/blob/main/src/Checks/Checks/QueueCheck.php
